source/manual/alerts/search-reindex-failed.html.md

```diff
@@ -0,0 +1,17 @@
+---
+owner_slack: "#2ndline"
+title: Search reindex failed
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-11-23
+review_in: 12 months
+---
+
+The reindex task is run weekly on a Monday at 12pm on integration.  This is to ensure the process workes as
+expected when we need to run it in production.
+
+This task is will need to be run in production in the event of the elasticsearch schema being changed.
+
+If this process fails then please escalate to the Search team as they are responsible for ensuring this process
```

Comment:
Is there anything that 2nd line can look into first? It would be good to give them some starting points in case the search team isn't around. Maybe check the Elasticsearch cluster health?

A link to the rake task might be useful too.

Reply:
I feel that we should take responsibility if this fails at this stage. We may want to change this once the process is working successfully, and at that point, we should update this document.

Until then we should be the ones being notified and looking into the issues.